examples/MvcWeb/Services/ProjectTypeService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.UnitOfWork;
+
+namespace MvcWeb.Services
+{
+    public class ProjectTypeService : IProjectTypeService
+    {
+        private readonly IUnitOfWork _uow;
+        public ProjectTypeService(IUnitOfWork unitOfWork)
+        {
+            _uow = unitOfWork;
+        }
+        public IEnumerable<ProjectType> GetAllProjectsType()
+        {
+            return _uow.Repository<ProjectType>().Get().AsNoTracking().ToList();
+        }
+
+        public ProjectType GetProjectTypeById(int id)
+        {
+            try
+            {
+                ProjectType projectType = _uow.Repository<ProjectType>().Get().FirstOrDefault(p => p.Id == id);
+                if (id == default(int))
+                {
+                    throw new ArgumentException($"{nameof(id)} cannot be null");
+                }
```

Comment:
just think,  what is the point of checking this here? after we already did call to the repo?
Also, the default of int is 0, not null.